app/models/orchestration_template.rb

```diff
@@ -70,6 +73,11 @@ def content=(text)
     self.md5 = calc_md5(content)
   end
 
+  # Determines if validation for md5 uniqueness is done
+  def unique_md5?
+    !draft? || !ems_ref
```

Comment:
This is particularly useful when a user adds a VNFD template. When a new template is added, there is no `ems_ref` so we will check md5 uniqueness. No two VNFDs with the same content can be added. However this also means a new VNFD cannot  be added if there is an existing one with `ems_ref`, which could be a problem. We may live with it for now,